config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "appid": "",
   "group": "",
-  "webscene": "19faa71a3bf6468cae35b4fce9393a7d",
+  "webscene": "",
```

Comment:
It was a tradeoff to permit running the boilerplate with ?webmap=*id*, ?webscene=*id*, ?group=*id* out of the box.  With the webscene id in config.json, running ?group=*id* brings up that webscene rather than the group.  It means that the boilerplate won't bring up a default display if no URL parameters are provided, but that seems more intuitive than having ?group=*id* bring up a webscene.